### PR TITLE
Connect via birthday

### PR DESCRIPTION
**Pull request by Crazymad**

### CODE REVIEW

File: config/services.yaml [hunk 1]
```diff
@@ -44,8 +44,4 @@ services:
     # please note that last definitions always *replace* previous ones
     App\Form\Type\UserType:
         arguments:
-            $availableSkillSets: '%app.available_skill_sets%'
-
-    App\Security\UserLoginFormAuthenticator:
-        arguments:
-            $password: 'covid19'
+            $availableSkillSets: '%app.available_skill_sets%'
```
Comment (mRoca) [inline config/services.yaml]: empty line is missing after this

File: src/Form/Type/UserType.php [hunk 2]
```diff
@@ -48,6 +48,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         $occupationChoices = (array) array_combine(self::DEFAULT_OCCUPATIONS, self::DEFAULT_OCCUPATIONS);
         $occupationChoices += ['Autre :' => '-'];
         $builder
+            ->add('identificationNumber', TextType::class)
```
Comment (kissifrot) [inline src/Form/Type/UserType.php]: Why here?

**Comment by mRoca** [comment]
Can you squash your commits ?

File: src/Form/Type/UserType.php (hunk 2 above)
Comment (kissifrot) [reply src/Form/Type/UserType.php]: ?

**Pull request merged**